Client/UIStackViewExtensions.swift

```diff
@@ -0,0 +1,51 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+extension UIStackView {
+
+    func toggleStackViewVisibility(show: Bool) {
```

Comment:
nit: to error proof some of these extension functions

Reply:
Will do some tests in next PR as well